.github/workflows/main.yml

```diff
@@ -10,58 +10,100 @@ jobs:
   build-package:
     name: "Build Package"
     runs-on: macos-latest
+    strategy:
+      matrix:
+        xcode:
+        - '12.4' # Swift 5.3
```

Comment:
Why do we need to support Swift 5.3? Just curious

Reply:
We don't have to, it was just previously supported in `master` based on https://swiftpackageindex.com/airbnb/lottie-ios so I continued to support it

Reply:
Good to know!